src/components/Button.jsx

```diff
@@ -1,3 +1,4 @@
-const Button = () => {};
-
+const Button = ({ color, text }) => {
+  return <button style={{ background: color }}> {text}</button>;
```

Comment:
use a div tag here, you are defining the button tag so you will get an error trying to use this. Also, the button doesn't need a color parameter, it should be the same for every button.